README.md

```diff
@@ -1,14 +1,12 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2223S1-CS2103-F09-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2223S1-CS2103-F09-2/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+# Duke-The-Market
+
+Duke-The-Market is a one-stop marketing tool that allows department stores to keep track of their upcoming marketing plan roll outs, monitor its impact, and to target the appropriate subsegment of its customer base for each of those plans.
+
+* If your are keen on using this application, head over to the User Guide [Link TBC]
+* If you are keen on adding new features to this application, head over to the Developer Guide [Link TBC]
+
+This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```

Comment:
I think you will need to add an extra newline character to the end of this file.

Reply:
Thanks for pointing out. Seems like there're more than 1 locations where the newline is causing failure. Since there are multiple issues here
1. naming convention of branch 
2. newline line issue for codecov at multiple locations
3. change of UI due to removal of purchase history & addition of events class

I'll just close this issue, and create a new branch after pulling the latest and fixing the above